src/components/shared/SearchInput.js

```diff
@@ -139,6 +139,8 @@ class SearchInput extends Component {
       onKeyUp,
       onFocus,
       onBlur,
+      "aria-autocomplete": "list",
+      "aria-controls": "result-list",
```

Comment:
aria-controls has to point to an id ref (e.g. other element's id attribute). Since you are referencing "result-list" here, it has to be set as an id for the result-list <UL>.